traffic_ops/testing/api/v1/crconfig_test.go

```diff
@@ -153,6 +155,89 @@ func UpdateTestCRConfigSnapshot(t *testing.T) {
 	}
 }
 
+func MonitoringConfig(t *testing.T) {
+	if len(testData.CDNs) < 1 {
+		t.Fatalf("no cdn test data")
+	}
+	const cdnName = "cdn1"
+	const profileName = "EDGE1"
+	cdns, _, err := TOSession.GetCDNByName(cdnName)
+	if err != nil {
+		t.Fatalf("getting CDNs with name '%s': %s", cdnName, err.Error())
+	}
+	if len(cdns) < 1 {
+		t.Fatalf("expected to find a CDN named %s", cdnName)
+	}
+	if len(cdns) > 1 {
+		t.Fatalf("expected exactly 1 CDN named %s but found %d CDNs", cdnName, len(cdns))
+	}
+	profiles, _, err := TOSession.GetProfileByName(profileName)
+	if err != nil {
+		t.Fatalf("getting Profiles with name '%s': %s", profileName, err.Error())
+	}
+	if len(cdns) < 1 {
+		t.Fatalf("expected to find a Profile named %s", profileName)
+	}
+	if len(cdns) > 1 {
+		t.Fatalf("expected exactly 1 Profiles named %s but found %d Profiles", profileName, len(profiles))
```

Comment:
I'm not sure why this didn't make it into my review, but it was supposed to:

This seems like it should be checking the length of `profiles`, not `cdns`.

Also, I think if you remove the `len(thing) < 1` blocks and change the `len(thing) > 1` blocks' conditions to `len(thing) != 1` the error message therein will suffice to cover both cases. It's too late to save some typing now, but I thought I'd point it out in case you wanna change it anyway.

Reply:
Checking the length of `profiles` in 3963215ba4.

Removed the redundant condition in d5387d1139.